### ISSUE TITLE
Would you consider removing the timelock in v0.3.03.20, but don't change anything in esrgan or bvpp?

### ISSUE THREAD

**Issue by teijiIshida** [km2ii/JVP_TG-PLUS#196]
Hello km2ii,

As stated in my [previous issue thread](https://github.com/km2ii/JVP_TG-PLUS/issues/194), I cannot get esrgan1 and 2 to work correctly with v.03.07.12 no matter what I try. esrgan is VERY IMPORTANT to me and is the only secondary enhancement I use due to its superior results. Would you consider removing the timelock in **v0.3.03.20_patcher(20230525)**, but don't change anything in esrgan or bvpp like in v.03.07.12? Version **v0.3.03.20_patcher(20230525)** was perfect for me. (You can add folder limit if you wish.)

I'll do a donation if version v0.3.03.20 is working again like before. Please please please.

**Comment by newborn2010** [comment]
+1
v0.3.03.20 is the most satisfying version so far.

**Comment by Jonethon-wei** [comment]
For different versions, I have tested different model combinations, but I think that v0.3.02.02, when the decoding area is set back to the original file, the stitching is relatively smooth, the color is more consistent, the flicker is much lower than the subsequent version, and it will not make the Mosaic clearer, so that multiple models make the part mixed with "hands" seriously deformed. The other Settings in config of v0.3.02.02 must set offset_x & offset_y to -2(the result of repeated test and comparison with more than 20 movies). My model combination is basicVSR-8x-std-ep1 & SE-4X-S1. Decoding speed is also relatively fast, but the quality is better than tecogan-da. If you still care about flicker, denoise-td-medium(or high) can be added.

**Comment by Jonethon-wei** [comment]
If you are still interested in v0.3.03.20, it is recommended that pacher update to 20230326, its effect is also good, there are also esrgan you like to use, and there is no time limit, but the processed film will appear a few similar words are very small long watermarks, although the whole film only appears less than 10 places. It doesn't affect the viewing, but I don't think so.

**Comment by Jonethon-wei** [comment]
I usually do not recommend the use of more than ep2 modules, usually it may make some parts clear, but other parts contained in the Maasque area may be seriously deformed, I think the decoded image is stitched to the film, the film itself needs to have the overall smoothness at the end, it will be more comfortable to watch.

**Comment by teijiIshida** [comment]
> 

Umm, it is not about the settings. The flicker in v.03.07.12's esrgan1/2 happens regardless of the settings you use, which it doesn't happen in v0.3.03.20. That's why my suggestion to the author to remove the timelock in the old version for people like me who can't use the new version because of instability in esrgan.

**Comment by Jonethon-wei** [comment]
I think the author is basically trying to prevent commercial use by unscrupulous people, but for those of us who are interested in researching, we will be limited by the software's time or the number of directories, so I think v0.3.02.02 is the version to choose. I've been testing various possible module combinations for the same preferred movie, through several subsequent versions, and finally found that for the movies I decoded between 2/20-3/10, it was almost difficult for the later versions to surpass its overall decoding quality, so I did a comparison between the v0.3.03.20 update to version 230326 and the v0.3.02.02 version, and chose to do it in the same module combination environment. Comparing the two versions, v0.3.02.022 performs better and doesn't have the annoying floating watermarks. Versions above ep2 tend to make the decoded blocks unnatural.

**Comment by km2ii** [comment]
I think testing and sharing 5-10 minute videos is enough for most people, rather than making and distributing ai-spam.

**Comment by wwallace55** [comment]
You can still use v0.3.03.20.  Just disconnect your internet and the annoying timelock goes away.  Putting in this timelock, limiting directories and adding watermarks are all a waste of time at this point since you can use v0.3.02.02 and any older versions.  This is literally a case of "closing the barn door after the horse is gone".

**Comment by newborn2010** [comment]
> You can still use v0.3.03.20. Just disconnect your internet and the annoying timelock goes away. Putting in this timelock, limiting directories and adding watermarks are all a waste of time at this point since you can use v0.3.02.02 and any older versions. This is literally a case of "closing the barn door after the horse is gone".

Yes, there is always a way to crack it. Software limitations just hurt normal users. Because the commercial websites don't care about the new functions and models of the software at all, as long as the old version of the software can be used. They just apply the simplest templates to mass-produce videos and then add their website watermarks.
===================================
Cracking methods of v0.3.02.02:
Create a windows firewall blocking rules of  \TG\TGMAIN\rungan.exe 
or 
add "78.46.102.180 rungan" in C:\Windows\System32\drivers\etc\hosts

**Comment by Jonethon-wei** [comment]
v0.3.03.20 Updated to 230326 no usage time not locked, after 230505 was asked to force update.

**Comment by Jonethon-wei** [comment]
v0.3.03.20 Updated to 230326 use time not locked, after 230505 was asked to force update.

**Comment by Jonethon-wei** [comment]
I have successfully setup "78.46.102.180 rungan" in C:\Windows\System32\drivers\etc\hosts, although the update screen pops up, the decryption will not be interrupted. Thank you newborn2010.

**Comment by wwallace55** [comment]
I used the Avast virus protection firewall on \TG\TGMAIN\rungan.exe to block all inbound and outbound traffic.  There's no more pop up window and JavPlayer is processing faster now as well.   Just another way to make it work.

**Comment by Jonethon-wei** [comment]
Use windows defender to set the in & out rules (both of them) to block the update message. If you change the hots alone, the update message will still appear after unlocking the mosaic program, so you have to close it before you can continue to run the recode.

**Comment by km2ii** [comment]
**Yes, you can continue to use the old version.**

**The mod has been maintained free of charge for 3 years, but has been maliciously peddled for profit since the beginning.
You have become accomplices to malicious sellers, especially by openly discussing cracks in the project's discussion forums.
I consider this an open provocation and have decided to terminate the maintenance of the project.**

**Have fun and goodbye.**
